src/graph/KokkosGraph_GraphColorHandle.hpp

```diff
@@ -60,8 +63,6 @@ enum ColoringAlgorithm { COLORING_DEFAULT,
                          COLORING_EB,                         // Edge Based Coloring
                          COLORING_SERIAL2,
                          COLORING_SPGEMM,
```

Comment:
This is distance2 right ? Can we move this to ditance2. What is SERIAL2, Can we have more descriptive for name ?

Reply:
The `COLORING_SPGEMM` should be the Distance-2 Coloring using SPGEMM + D1 Coloring. I don't think that's in the D1 coloring handle anymore so I'll remove this.

The `COLORING_SERIAL2` is still in the Distance1 coloring handle as an artifact from a while back. I think it's used in the SPGEMM code in a couple places so it's not quite so trivial to cut that out.  I'd like to move this over fully to the Distance2 handle. Maybe it might be good to file an issue and do that part as a separate PR so we can get the Distance2 coloring in since there are some bigger changes with interfaces coming in with this?